### PR TITLE
build(aio): make less verbose tools

### PR DESCRIPTION
**Pull request by Foxandxss**
@petebacondarwin asked for the tools to be less verbose somewhere (is there an issue for that?)

I left some as a comment because I plan on revisit them.

### CODE REVIEW

File: aio/tools/plunker-builder/builder.js [hunk 1]
```diff
@@ -232,10 +232,10 @@ class PlunkerBuilder {
       // console.log('** Substituted "' + webExtras + '"  for "' + extras + '".');
       return fs.readFileSync(webExtras, 'utf-8');
     } else if (this._existsSync(extras)){
-      console.log('** WARNING: no "' + webExtras + '" replacement for "' + extras + '".');
+      // console.log('** WARNING: no "' + webExtras + '" replacement for "' + extras + '".');
```
Comment (petebacondarwin) [inline aio/tools/plunker-builder/builder.js]: let's keep the warnings

**Comment by Foxandxss** [comment]
Done.

**Review by petebacondarwin — APPROVED** [review]

File: aio/tools/plunker-builder/builder.js (hunk 1 above)
Comment (gkalpak) [inline aio/tools/plunker-builder/builder.js]: @petebacondarwin, why?
@Foxandxss, what are these warnings for? Are missing something we need to fix or is it normal for some of them to be missing?

Comment (petebacondarwin) [inline aio/tools/plunker-builder/builder.js]: Exactly. If they are not actionable then they are just informative and can go.

**Pull request merged**